modules/tinymce/src/core/main/ts/DragDropOverrides.ts

```diff
@@ -55,7 +55,7 @@ const isValidDropTarget = (editor: Editor, targetElement: Node | null, dragEleme
   } else if (targetElement === dragElement || editor.dom.isChildOf(targetElement, dragElement)) {
     return false;
   } else {
-    return !isContentEditableFalse(targetElement);
+    return !isContentEditableFalse(targetElement) && editor.dom.getContentEditableParent(targetElement) !== 'false';
```

Comment:
Is the intention here that we should support dropping onto a contenteditable=true element that is within a contenteditable false element? If so, should we add a test for that scenario?

Reply:
The intention is to prevent dropping onto a descendant of a noneditable element.

Reply:
I added a test for dropping onto a contenteditable=true element that is within a contenteditable=false element.